packages/amplify_datastore/test/amplify_datastore_query_test.dart

```diff
@@ -49,7 +49,9 @@ void main() {
         Comment(
             id: '39c3c0e6-8726-436e-8cdf-bff38e9a62da',
             content: 'Loving Amplify Datastore!',
-            post: Post(id: 'e50ffa8f-783b-4780-89b4-27043ffc35be')));
+            post: Post(
+                id: 'e50ffa8f-783b-4780-89b4-27043ffc35be',
+                title: ""))); // TODO POSSIBLE ISSUE HERE CONSTRUCTOR ID ONLY
```

Comment:
Do we still need this comment?

Reply:
no I'll remove that

Reply:
Nit: Since this is test data and we expect Title to be non-null.. maybe just give this a meaningful string?

Reply:
sounds good, update title to "some title"